src/styles/utils/reset.scss

```diff
@@ -0,0 +1,6 @@
+*,
+p {
+  margin: 0;
+  padding: 0;
+  justify-content: center;

```

Comment:
It is not necessary to set the flex property without the flex itself to all elements, this is an unnecessary overload of styles.